modules/createRenderer.js

```diff
@@ -15,12 +15,18 @@ import cssifyObject from './utils/cssifyObject'
  * @return {Object} new renderer instance
  */
 export default function createRenderer(config = { }) {
+  const prettySelectors = config.prettySelectors || false
```

Comment:
You can remove this and use `config.prettySelectors` directly as if undefined it is falsy by default